### PR TITLE
[ndslice]: doc fix follow-up

### PR DESCRIPTION
**Pull request by wilzbach**
@9il 
#4237 forgot the changelog entry - moreover I added the grammatic fix pointed out by @tsbockman and reindented the lines (it was > 120).

### CODE REVIEW

File: std/experimental/ndslice/slice.d [hunk 1]
```diff
@@ -1180,13 +1180,16 @@ struct Slice(size_t _N, _Range)
...
-        plain structure with two fields `shift` and `range` otherwise. In second case the expression `range[shift]`
-        refers to the first element. For slices with named elements the type of a return value has the same behavior like a pointer.
+        Pointer to the first element of a slice if slice is defined as `Slice!(N, T*)`
+        or plain structure with two fields `shift` and `range` otherwise.
+        In second case the expression `range[shift]` refers to the first element.
+        For slices with named elements the type of a return value
+        has the same behavior like a pointer.
     Note:
-        `ptr` is defined only for not packed slices.
+        `ptr` is defined only for non-packed slices.
```
Comment (9il) [inline std/experimental/ndslice/slice.d]: thx)

File: changelog.dd [hunk 2]
```diff
@@ -29,6 +29,9 @@ $(BUGSTITLE Library Changes,
         rangified.)
     $(LI All overloads of `std.conv.toImpl` has been made private. Please use
         $(XREF conv, to) instead.)
+    $(LI `std.experimental.ndslice`: For non-packed slices the property `ptr` was
```
Comment (9il) [inline changelog.dd]: Could you please remove this. I will add more detailed description?

**Comment by wilzbach** [comment]
Okay rebased ;-)

**Comment by 9il** [comment]
Auto-merge toggled on

**Comment by wilzbach** [comment]
```
core.exception.AssertError@std/file.d(4022): unittest failure
----------------
??:? _d_unittest [0x537145c]
```

what?

btw @9il i guess it's better to remove the decision block label, so people don't wonder.

**Pull request merged**